llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -712,13 +793,19 @@ int main(int argc, char **argv) {
       "This is a collection of utilities run on device code's LLVM IR before\n"
       "handing off to back-end for further compilation or emitting SPIRV.\n"
       "The utilities are:\n"
+      "- SYCL and ESIMD kernels can be split into separate modules with\n"
+      "  '-split-esimd' option. The option has no effect when there is only\n"
+      "  one type of kernels in the input module.\n"
```

Comment:
Suggest to add:
Functions unreachable from any kernel (including those marked as with "explicit SIMD" metadata) are dropped from the resulting module(s).

Reply:
Added